packages/hash/frontend/src/blocks/page/createSuggester/useFilteredBlocks.tsx

```diff
@@ -10,25 +10,40 @@ type Option = {
   meta: BlockMeta["componentMetadata"];
 };
 
+const SWAPPABLE_BLOCKS = [
+  "https://blockprotocol.org/blocks/@hash/paragraph",
+  "https://blockprotocol.org/blocks/@hash/header",
+  "https://blockprotocol.org/blocks/@hash/callout",
+];
+
+export const isBlockSwappable = (blockId: string = "") =>
+  SWAPPABLE_BLOCKS.includes(blockId);
+
 export const useFilteredBlocks = (
   searchText: string,
   blocksMetaMap: BlocksMetaMap,
+  textBlocksOnly = false,
```

Comment:
We can potentially replace this with something like:

```suggestion
  { textBlocksOnly = false }: { textBlocksOnly?: boolean } = {},
```

Alternatively, we can switch to all named args. Current function signature requires folks to hover over the third arg to understand its meaning: `useFilteredBlocks(searchQuery, userBlocks, true)`

Reply:
This was talked about in this [thread](https://hashintel.slack.com/archives/C02TWBTT3ED/p1657633845727149) that @nathggns created last week. 
I'm happy to make this change as I don't really have a strong opinion on this but, from the thread, I don't think the team is leaning on this pattern for defining args.

Reply:
I'm still not convinced that `doX(true)` is a good coding pattern because far not all editors can hint on what `true` is. But I don't think it's worth debating because we're not talking about a public API.